### DISCUSSION TITLE
Podman 4.2 and Podman Desktop System Requirements for Mac and Windows

### DISCUSSION THREAD

**Discussion by dshea18** [containers/podman#15702] [Q&A] [answered]
What are the bare minimum system requirements for Podman v4.2.0 and Podman Desktop for both MacOS and Windows?

Thanks in advance,
Daniel Shea

**Comment by mheon** [comment]
Podman requirements are mostly around kernel version - the oldest kernel we do regular testing with is the 4.18 kernel of RHEL8. Our resource requirements otherwise are quite minimal. On OS X and Windows, the `podman machine` VM will automatically take care of this.

For Podman Desktop, I will tag @benoitf

**Reply by dshea18** [reply]
Thank you.

**Reply by dshea18** [reply]
Are there any MacOS version requirements?

**Reply by rhatdan** [reply]
I think Mac 4.14?

**Comment by benoitf** [comment, marked as answer]
hello, for Podman Desktop 

Requirements are 2 CPU, 4GB of Ram and macOS Catalina or more r ecent for macOS

For Windows, WSL2/Hyper-v, 6GB of RAM and Windows 10 Build 18362 or later

**Reply by dshea18** [reply]
Thank you.
